package.json

```diff
@@ -52,7 +52,8 @@
     "react-dom": "^17.0.0",
     "react-helmet": "^6.1.0",
     "react-scroll": "^1.8.7",
-    "react-spring": "9.1.2",
```

Comment:
i initially upgraded `react-spring` bc newer versions support interpolating CSS vars. unfortunately, this only works if the var is declared on `:root`, so this didn't end up mattering. doesn't hurt to upgrade though